docs/src/components/Navbar/Links.tsx

```diff
@@ -9,7 +9,7 @@ import ReadArt from "../../images/streamline-light/read-art.svg";
 import IrisScanIcon from "../../images/streamline-light/iris-scan.svg";
 import ScriptIcon from "../../images/streamline-light/script.svg";
 
-type Link = "Components" | "Guides" | "Tokens" | "Accessibility" | "Foundation" | "Playground";
```

Comment:
Are we removing the playground logic? If so, will it be part of a different task?

In case we want to create a ticket.

Reply:
tbh, there was no decision about that set, I've changed it "naturally", because the functionality is very similar.